sql/moz-fx-data-shared-prod/braze_derived/products_v1/query.sql

```diff
@@ -0,0 +1,22 @@
+SELECT
+  fxa_uid AS fxa_id_sha256,
```

Comment:
Do we have a way of mapping fxa_id to email_id? Braze won't know what to do with fxa_id by itself.

Reply:
we do! there's a table in ctms that will remain and it holds just email_id and fxa_id. i don't see why we would need to bring in fxa_id at all tbh. i can't imagine it's being used in braze...it's in the planning sheet so I was bringing it in...but if we can leave it out entirely, that'd be preferable